doc/reference_processing.xml

```diff
@@ -1787,10 +1802,12 @@ POINT(2 1)
 
 		<para>
 			ST_GeneratePoints generates pseudo-random points until the requested number are
-			found within the input area.
+			found within the input area. The seed parameter allows re-generation of
+			the same set of points.
```

Comment:
I'm hesistant to give guarantees in docs. If we tweak point-in-poly a bit the set may be different between postgis versions, or maybe even platforms, even with same random functions - think JIT and 64/80-bit maths. 

Maybe "setting seed allows Postgres to cache results between runs"?

Reply:
I'm confident that on one computer that point sequence can be re-generated with a user-supplied seed. I'll reword to not imply absolute re-generation of floating-point values across different hardware.

Reply:
Well I did reword this part, but I can't think of a simple way to not over complicate the docs. Pseudorandom number generators are deterministic by design. I'm just not mentioning any guarantees of generating the exact sequence of points across all hardware and software versions.